ansible/vagrant/Vagrantfile

```diff
@@ -137,6 +137,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       ansible.groups = groups
       ansible.playbook = "../cluster.yml"
       ansible.limit = "all" #otherwise the metadata wont be there for ipv4?
+      # Example variables which we might set.
+      #		iface: This is for binding flannel to the non-default eth device.
+      #		apiserver_runtime_config: This can be provided to, for example, enable deployments (hpa).
+      # ansible.extra_vars = { flannel_iface: "eth1", apiserver_runtime_config: "extensions/v1beta1/deployments=true" }
```

Comment:
Do you need to just set `flannel_opts` on vbox? (it's not called `flannel_iface` and more)

`apiserver_runtime_config` seems better documented out in ansible (maybe a commented out example in the master/defaults/meta.yaml) and not here in Vagrant stuff.

Should (and unset) flannel_opts be out in the ansible repo?